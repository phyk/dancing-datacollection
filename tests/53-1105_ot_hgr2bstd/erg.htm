<!DOCTYPE html PUBLIC "-//WAPFORUM//DTD XHTML Mobile 1.2//EN" "http://www.openmobilealliance.org/tech/DTD/xhtml-mobile12.dtd">
<HTML>
<HEAD>
  <META http-equiv="Content-Type" content="text/html; charset=utf-8">
  <META name="Author" content="Hessischer Tanzsportverband e.V.">
  <META name="GENERATOR" content="TopTurnierDigital">
  <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
  <meta http-equiv="Pragma" content="no-cache" />
  <meta http-equiv="Expires" content="0" />
  <TITLE>11.05.2024 Hgr.II B Standard</TITLE>
  <link rel="stylesheet" type="text/css" href="topturnier.css">
  <meta name="viewport" content="width=device-width, initial-scale=1" />
</HEAD>
<body><div class="mainback">
<div class="eventhead">
<table border=0 width=100%>
<tr><td>11.05.2024 - OT, Hgr.II B Standard &#34;2&#34;</td><td width=30>
<a class="backbtn" href="index.htm" target="_top">&equiv;</a>
</td></tr>
</table>
</div>
<div class="maincontainer">
<div class="comphead">Ergebnis</div>
<hr class="line">
<div class="extract">
   <TABLE class="tab1">
    <TR>
     <TD class="td1" colspan="8">Endrunde</TD>
    </TR>
    <TR>
     <TD class="td2c" width="45px">Platz</TD>
     <TD class="td2">Paar/Club</TD>
     <TD class="td2cw" align=center nowrap>LW</TD>
     <TD class="td2cw" align=center nowrap>TG</TD>
     <TD class="td2cw" align=center nowrap>WW</TD>
     <TD class="td2cw" align=center nowrap>SF</TD>
     <TD class="td2cw" align=center nowrap>QU</TD>
     <TD class="td2" align=center nowrap>PZ</TD>
    </TR>
    <TR>
     <TD class="td3r">1.</TD>
     <TD class="td5">Maximilian Beichter  /  Melissa Hagel (401)<BR><i>TSC Astoria Karlsruhe</i></TD>
     <TD class="td3w">2111114<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">2211213<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">2211213<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">1111212<BR><div class="pz">1.0</div></TD>
     <TD class="td3w">2211212<BR><div class="pz">1.0</div></TD>
     <TD class="td3pz">&nbsp;<br> 5.0</TD>
    </TR>
    <TR>
     <TD class="td3r">2.</TD>
     <TD class="td5">Leif-Erik Montag  /  Johanna Wille (421)<BR><i>Tanzsportteam im  ASC G&ouml;ttingen v. 1846</i></TD>
     <TD class="td3w">1244331<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">1135341<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">1144521<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">2244321<BR><div class="pz">2.0</div></TD>
     <TD class="td3w">1134331<BR><div class="pz">2.0</div></TD>
     <TD class="td3pz">&nbsp;<br>10.0</TD>
    </TR>
    <TR>
     <TD class="td3r">3.</TD>
     <TD class="td5">Marcus Nguyen Ngoc  /  Lea Te&szlig;mer (423)<BR><i>Club C&#233;ronne im ETV Hamburg</i></TD>
     <TD class="td3w">6362245<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">5352135<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">6363136<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">6363145<BR><div class="pz">3.0</div></TD>
     <TD class="td3w">5352125<BR><div class="pz">3.0</div></TD>
     <TD class="td3pz">&nbsp;<br>15.0</TD>
    </TR>
    <TR>
     <TD class="td3r">4.</TD>
     <TD class="td5">Patrick Dahm  /  Sandra Schwarz (408)<BR><i>TC Der Frankfurter Kreis</i></TD>
     <TD class="td3w">5553423<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">4563424<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">5552342<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">4552533<BR><div class="pz">4.0</div></TD>
     <TD class="td3w">4563444<BR><div class="pz">4.0</div></TD>
     <TD class="td3pz">&nbsp;<br>20.0</TD>
    </TR>
    <TR>
     <TD class="td3r">5.</TD>
     <TD class="td5">Christopher Buchloh-Rosenthal  /  Analena Koch (405)<BR><i>Rot-Weiss-Klub Kassel</i></TD>
     <TD class="td3w">3435556<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">3446566<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">3435455<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">3436456<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">3426556<BR><div class="pz">5.0</div></TD>
     <TD class="td3pz">&nbsp;<br>26.0</TD>
    </TR>
    <TR>
     <TD class="td3r">6.</TD>
     <TD class="td5">Jakob Hinz  /  Vivien Bachmann (415)<BR><i>TC Kristall Jena</i></TD>
     <TD class="td3w">4626662<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">6624652<BR><div class="pz">5.0</div></TD>
     <TD class="td3w">4626664<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">5625664<BR><div class="pz">6.0</div></TD>
     <TD class="td3w">6645663<BR><div class="pz">6.0</div></TD>
     <TD class="td3pz">&nbsp;<br>29.0</TD>
    </TR>
   </TABLE>
   <TABLE class="tab2" border=0>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>2. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">7.</TD>
     <TD class="td5">Mariusz Budek  /  Marta Budek (406)</TD>
     <TD class="td6">TSC Villingen-Schwenningen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">8.</TD>
     <TD class="td5">Christoph Hanisch  /  Kaja Zo&#233; Pf&uuml;ller (412)</TD>
     <TD class="td6">UTSC Choice Styria</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">9.</TD>
     <TD class="td5">Sebastian Hellmann  /  Melanie Oberhauser (414)</TD>
     <TD class="td6">TTC Oldenburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">10.</TD>
     <TD class="td5">Oliver Neumann  /  Anna-Maria Ehinger (422)</TD>
     <TD class="td6">TSC Astoria Karlsruhe</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">11.- 12.</TD>
     <TD class="td5">Ruslan Wellner  /  Tabea Kilian (430)</TD>
     <TD class="td6">Braunschweig Dance Company</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">11.- 12.</TD>
     <TD class="td5">Leo Werner  /  Fabienne Theobalt (432)</TD>
     <TD class="td6">TC Rot-Weiss Casino Mainz</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>1. Zwischenrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">13.- 14.</TD>
     <TD class="td5">Uli Kunz  /  Saskia Morcinczyk (420)</TD>
     <TD class="td6">TSC Gr&uuml;n-Gold Speyer</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">13.- 14.</TD>
     <TD class="td5">Martin Wenhart  /  Lisa Harrell (431)</TD>
     <TD class="td6">TSC dancepoint, K&ouml;nigsbrunn</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">15.</TD>
     <TD class="td5">Ren&#233; Kaczorowski  /  Cindy Hebert (417)</TD>
     <TD class="td6">Tanzsportverein Schwarz-Wei&szlig; Freiberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">16.</TD>
     <TD class="td5">Benedikt Ernst  /  Tanja Esche (411)</TD>
     <TD class="td6">TSC Rot-Gold-Casino N&uuml;rnberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">17.</TD>
     <TD class="td5">Fabian Beckmann  /  Katrin Langert (400)</TD>
     <TD class="td6">TSC Schwarz-Gelb Aachen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">18.- 19.</TD>
     <TD class="td5">Patrick Rach  /  Lorena Kimmel (427)</TD>
     <TD class="td6">TC Rot-Weiss Casino Mainz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">18.- 19.</TD>
     <TD class="td5">Enrico Weber  /  Anne-Kathrin Nitt-Weber (429)</TD>
     <TD class="td6">Tanzsportzentrum Dresden</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">20.</TD>
     <TD class="td5">Max Kirchenberger  /  Friederike Rust (418)</TD>
     <TD class="td6">TTC M&uuml;nchen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">21.- 22.</TD>
     <TD class="td5">Johannes Kreim  /  Rebecca Gonzalez-Ringer (419)</TD>
     <TD class="td6">TC Rot-Weiss Casino Mainz</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">21.- 22.</TD>
     <TD class="td5">Patrik Pollak  /  Pia Feischen (426)</TD>
     <TD class="td6">TSC Gr&uuml;n-Gold Heidelberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">23.- 24.</TD>
     <TD class="td5">Christopher Brix  /  Sandra Kuhfus (403)</TD>
     <TD class="td6">TSC Gr&uuml;n-Wei&szlig; Aquisgrana Aachen</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">23.- 24.</TD>
     <TD class="td5">Michael Wrusch  /  Dan Feng Tian Helena Liang (434)</TD>
     <TD class="td6">OTK Schwarz-Wei&szlig; 1922 im SCS Berlin</TD>
    </TR>
    <TR style="height: 5px; border:0;">
     <TD class="td5" colspan=3 style="height: 5px; border:0;"></TD>
    </TR>
    <TR>
     <TD class="td1" colspan=3>Vorrunde</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">25.- 26.</TD>
     <TD class="td5">Dr. Michel Oelschl&auml;gel  /  Dagmar Lina Ganz (424)</TD>
     <TD class="td6">Tanzsportverein Schwarz-Wei&szlig; Freiberg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">25.- 26.</TD>
     <TD class="td5">Matty Schiller  /  Anne Wienhold (428)</TD>
     <TD class="td6">TSA d. TTC Allround Rostock</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">27.- 28.</TD>
     <TD class="td5">Christoph Hellings  /  Maria Korosteleva (413)</TD>
     <TD class="td6">Switzerland</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">27.- 28.</TD>
     <TD class="td5">Michael Hopf  /  Iris Hopf (416)</TD>
     <TD class="td6">TSC Unterschlei&szlig;heim</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">29.</TD>
     <TD class="td5">Thomas Brunnengr&auml;ber  /  Mirjam Tittlus (404)</TD>
     <TD class="td6">TC Blau-Orange Wiesbaden</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">30.</TD>
     <TD class="td5">Robert Podgajny  /  Olesya Oshchepkova (425)</TD>
     <TD class="td6">TTC Rot-Wei&szlig; Freiburg</TD>
    </TR>
    <TR>
     <TD class="td3r" width="45px">31.</TD>
     <TD class="td5">Florian Dammeyer  /  Yasmin Deborah Gers (409)</TD>
     <TD class="td6">Die Residenz M&uuml;nster</TD>
    </TR>
   </TABLE>
<br>
</div>
</div>
</div>
  <P><FONT size="1" face="Arial">Diese Liste wurde mit <A TARGET="_blank" HREF="http://www.TopTurnier.de">TopTurnier f&uuml;r Windows V9.6b</A> erstellt.<br></FONT>
</body>
</HTML>
