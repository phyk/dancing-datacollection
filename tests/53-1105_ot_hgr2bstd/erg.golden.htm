<!DOCTYPE html>
<html>
 <head>
  <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
  <title>
   11.05.2024 Hgr.II B Standard
  </title>
 </head>
 <body>
  <div>
   <div>
    <table>
     <tr>
      <td>
       11.05.2024 - OT, Hgr.II B Standard "2"
      </td>
     </tr>
    </table>
   </div>
   <div>
    <div>
     Ergebnis
    </div>
    <hr/>
    <div>
     <table>
      <tr>
       <td>
        Endrunde
       </td>
      </tr>
      <tr>
       <td>
        Platz
       </td>
       <td>
        Paar/Club
       </td>
       <td>
        LW
       </td>
       <td>
        TG
       </td>
       <td>
        WW
       </td>
       <td>
        SF
       </td>
       <td>
        QU
       </td>
       <td>
        PZ
       </td>
      </tr>
      <tr>
       <td>
        1.
       </td>
       <td>
        Maximilian Beichter / Melissa Hagel (401)
        <br/>
        <i>
         TSC Astoria Karlsruhe
        </i>
       </td>
       <td>
        2111114
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        2211213
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        2211213
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        1111212
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        2211212
        <br/>
        <div>
         1.0
        </div>
       </td>
       <td>
        <br/>
        5.0
       </td>
      </tr>
      <tr>
       <td>
        2.
       </td>
       <td>
        Leif-Erik Montag / Johanna Wille (421)
        <br/>
        <i>
         Tanzsportteam im ASC Göttingen v. 1846
        </i>
       </td>
       <td>
        1244331
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        1135341
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        1144521
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        2244321
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        1134331
        <br/>
        <div>
         2.0
        </div>
       </td>
       <td>
        <br/>
        10.0
       </td>
      </tr>
      <tr>
       <td>
        3.
       </td>
       <td>
        Marcus Nguyen Ngoc / Lea Teßmer (423)
        <br/>
        <i>
         Club Céronne im ETV Hamburg
        </i>
       </td>
       <td>
        6362245
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        5352135
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        6363136
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        6363145
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        5352125
        <br/>
        <div>
         3.0
        </div>
       </td>
       <td>
        <br/>
        15.0
       </td>
      </tr>
      <tr>
       <td>
        4.
       </td>
       <td>
        Patrick Dahm / Sandra Schwarz (408)
        <br/>
        <i>
         TC Der Frankfurter Kreis
        </i>
       </td>
       <td>
        5553423
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        4563424
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        5552342
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        4552533
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        4563444
        <br/>
        <div>
         4.0
        </div>
       </td>
       <td>
        <br/>
        20.0
       </td>
      </tr>
      <tr>
       <td>
        5.
       </td>
       <td>
        Christopher Buchloh-Rosenthal / Analena Koch (405)
        <br/>
        <i>
         Rot-Weiss-Klub Kassel
        </i>
       </td>
       <td>
        3435556
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        3446566
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        3435455
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        3436456
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        3426556
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        <br/>
        26.0
       </td>
      </tr>
      <tr>
       <td>
        6.
       </td>
       <td>
        Jakob Hinz / Vivien Bachmann (415)
        <br/>
        <i>
         TC Kristall Jena
        </i>
       </td>
       <td>
        4626662
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        6624652
        <br/>
        <div>
         5.0
        </div>
       </td>
       <td>
        4626664
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        5625664
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        6645663
        <br/>
        <div>
         6.0
        </div>
       </td>
       <td>
        <br/>
        29.0
       </td>
      </tr>
     </table>
     <table>
      <tr>
       <td>
        2. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        7.
       </td>
       <td>
        Mariusz Budek / Marta Budek (406)
       </td>
       <td>
        TSC Villingen-Schwenningen
       </td>
      </tr>
      <tr>
       <td>
        8.
       </td>
       <td>
        Christoph Hanisch / Kaja Zoé Pfüller (412)
       </td>
       <td>
        UTSC Choice Styria
       </td>
      </tr>
      <tr>
       <td>
        9.
       </td>
       <td>
        Sebastian Hellmann / Melanie Oberhauser (414)
       </td>
       <td>
        TTC Oldenburg
       </td>
      </tr>
      <tr>
       <td>
        10.
       </td>
       <td>
        Oliver Neumann / Anna-Maria Ehinger (422)
       </td>
       <td>
        TSC Astoria Karlsruhe
       </td>
      </tr>
      <tr>
       <td>
        11.- 12.
       </td>
       <td>
        Ruslan Wellner / Tabea Kilian (430)
       </td>
       <td>
        Braunschweig Dance Company
       </td>
      </tr>
      <tr>
       <td>
        11.- 12.
       </td>
       <td>
        Leo Werner / Fabienne Theobalt (432)
       </td>
       <td>
        TC Rot-Weiss Casino Mainz
       </td>
      </tr>
      <tr>
       <td>
        1. Zwischenrunde
       </td>
      </tr>
      <tr>
       <td>
        13.- 14.
       </td>
       <td>
        Uli Kunz / Saskia Morcinczyk (420)
       </td>
       <td>
        TSC Grün-Gold Speyer
       </td>
      </tr>
      <tr>
       <td>
        13.- 14.
       </td>
       <td>
        Martin Wenhart / Lisa Harrell (431)
       </td>
       <td>
        TSC dancepoint, Königsbrunn
       </td>
      </tr>
      <tr>
       <td>
        15.
       </td>
       <td>
        René Kaczorowski / Cindy Hebert (417)
       </td>
       <td>
        Tanzsportverein Schwarz-Weiß Freiberg
       </td>
      </tr>
      <tr>
       <td>
        16.
       </td>
       <td>
        Benedikt Ernst / Tanja Esche (411)
       </td>
       <td>
        TSC Rot-Gold-Casino Nürnberg
       </td>
      </tr>
      <tr>
       <td>
        17.
       </td>
       <td>
        Fabian Beckmann / Katrin Langert (400)
       </td>
       <td>
        TSC Schwarz-Gelb Aachen
       </td>
      </tr>
      <tr>
       <td>
        18.- 19.
       </td>
       <td>
        Patrick Rach / Lorena Kimmel (427)
       </td>
       <td>
        TC Rot-Weiss Casino Mainz
       </td>
      </tr>
      <tr>
       <td>
        18.- 19.
       </td>
       <td>
        Enrico Weber / Anne-Kathrin Nitt-Weber (429)
       </td>
       <td>
        Tanzsportzentrum Dresden
       </td>
      </tr>
      <tr>
       <td>
        20.
       </td>
       <td>
        Max Kirchenberger / Friederike Rust (418)
       </td>
       <td>
        TTC München
       </td>
      </tr>
      <tr>
       <td>
        21.- 22.
       </td>
       <td>
        Johannes Kreim / Rebecca Gonzalez-Ringer (419)
       </td>
       <td>
        TC Rot-Weiss Casino Mainz
       </td>
      </tr>
      <tr>
       <td>
        21.- 22.
       </td>
       <td>
        Patrik Pollak / Pia Feischen (426)
       </td>
       <td>
        TSC Grün-Gold Heidelberg
       </td>
      </tr>
      <tr>
       <td>
        23.- 24.
       </td>
       <td>
        Christopher Brix / Sandra Kuhfus (403)
       </td>
       <td>
        TSC Grün-Weiß Aquisgrana Aachen
       </td>
      </tr>
      <tr>
       <td>
        23.- 24.
       </td>
       <td>
        Michael Wrusch / Dan Feng Tian Helena Liang (434)
       </td>
       <td>
        OTK Schwarz-Weiß 1922 im SCS Berlin
       </td>
      </tr>
      <tr>
       <td>
        Vorrunde
       </td>
      </tr>
      <tr>
       <td>
        25.- 26.
       </td>
       <td>
        Dr. Michel Oelschlägel / Dagmar Lina Ganz (424)
       </td>
       <td>
        Tanzsportverein Schwarz-Weiß Freiberg
       </td>
      </tr>
      <tr>
       <td>
        25.- 26.
       </td>
       <td>
        Matty Schiller / Anne Wienhold (428)
       </td>
       <td>
        TSA d. TTC Allround Rostock
       </td>
      </tr>
      <tr>
       <td>
        27.- 28.
       </td>
       <td>
        Christoph Hellings / Maria Korosteleva (413)
       </td>
       <td>
        Switzerland
       </td>
      </tr>
      <tr>
       <td>
        27.- 28.
       </td>
       <td>
        Michael Hopf / Iris Hopf (416)
       </td>
       <td>
        TSC Unterschleißheim
       </td>
      </tr>
      <tr>
       <td>
        29.
       </td>
       <td>
        Thomas Brunnengräber / Mirjam Tittlus (404)
       </td>
       <td>
        TC Blau-Orange Wiesbaden
       </td>
      </tr>
      <tr>
       <td>
        30.
       </td>
       <td>
        Robert Podgajny / Olesya Oshchepkova (425)
       </td>
       <td>
        TTC Rot-Weiß Freiburg
       </td>
      </tr>
      <tr>
       <td>
        31.
       </td>
       <td>
        Florian Dammeyer / Yasmin Deborah Gers (409)
       </td>
       <td>
        Die Residenz Münster
       </td>
      </tr>
     </table>
     <br/>
    </div>
   </div>
  </div>
 </body>
</html>
